### ISSUE TITLE
Rebuilding an adventure with v10 scenes in v11 throws an error

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt#9754]
Originally reported by v3rotti https://discord.com/channels/170995199584108546/1065764070680186930/1126519253684723772

**ALL MODULES DISABLED?** true
**OS, Hosting, Browser (if applicable)**: The Forge, Chrome
**Short Description of bug**: When i load the application or try to rebuild an adventure (preexisting), i get this error. On load there are no issues but the Adventure Dialogue is stuck and could not be closed. Only refreshing the page helps to get the window closed, but the adventure is not updated. 
The Scenes and tokens where created in v10
**Simple steps to reproduce the bug**: Reloading Foundry or rebuilding an adventure
**Screenshots and/or console errors**: attached and

```
Error: Failed to initialize Token [s25fUi3VRGpY418J] in Scene [W2AUT3uJjadmSYeL]: TokenDocument5e [s25fUi3VRGpY418J] validation errors:
  delta: 
    items: 
      b9metMF1FcYuKrnY: 
        effects: 
          p34os5ydabfl94p0: 
            name: may not be undefined
      Pk7p1uXJfiSy3gfK: 
        effects: 
          s72pmqeg6b1txe7t: 
            name: may not be undefined
    at DataModelValidationFailure.asError (commons.js:3805:14)
    at TokenDocument5e.validate (commons.js:7067:61)
    at new DataModel (commons.js:6821:12)
    at new Document (commons.js:7484:3)
    at new BaseToken (commons.js:12163:3)
    at new ClientDocumentMixin (foundry.js:13950:7)
    at new CanvasDocument (foundry.js:13284:10)
    at new TokenDocument (foundry.js:24613:1)
    at new TokenDocument5e (token.mjs:6:16)
    at EmbeddedCollection.createDocument (commons.js:4323:14)
```

![image](https://github.com/foundryvtt/foundryvtt/assets/207433/f4d748a7-7c60-45da-91c8-31f7d202ab72)

![image](https://github.com/foundryvtt/foundryvtt/assets/207433/9be53454-186b-4cb9-9da1-8089138afe53)

![image](https://github.com/foundryvtt/foundryvtt/assets/207433/5b929774-43b3-4ee1-be54-4c453fccae60)

**Comment by Fyorl** [comment]
I suspect this is fixed by the resolution to #9786 but the reporter did not provide me their adventure compendium to test so I haven't been able to verify.

**Comment by Fyorl** [comment]
Closing as nonrepro.
